R/import_gtfs.R

```diff
@@ -187,9 +187,14 @@ import_gtfs <- function(path,
     quiet
   )
 
-  # assign names to 'gtfs'
-
-  names(gtfs) <- files_to_read
+  # assign names to 'gtfs', noting that zip_list may return full paths, which
+  # need to be stripped here
+
+  file_names <- vapply (files_in_gtfs, function (i)
```

Comment:
The `vapply()` should be applied to `files_to_read`, and not `files_in_gtfs` (a file may be inside the GTFS but the user may want to skip it using the `files` or `skip` parameters). This is probably why the PR didn't pass the checks.